selvpcclient/resell/v2/tokens/requests_opts.go

```diff
@@ -5,6 +5,6 @@ type TokenOpts struct {
 	// ProjectID can be used to create a project-scoped Identity token.
 	ProjectID string `json:"project_id,omitempty"`
 
-	// DomainName can be used to create a domain-scoped Identity token.
-	DomainName string `json:"domain_name,omitempty"`
+	// AccountName can be used to create a account-scoped Identity token.
```

Comment:
"AccountName can be used to create a domain-scoped Identity token."
We should follow keystone's naming here.